tests/project_test.py

```diff
@@ -87,7 +87,7 @@ def crew(self):
 
 @CrewBase
 class InternalCrewWithMCP(InternalCrew):
-    mcp_server_params = {"host": "localhost", "port": 8000}
+    mcp_server_params = [{"url": "localhost", "port": 8000}]
```

Comment:
Currently, we support defining MCP server parameters using a single dictionary.

With your change, users with that configuration might run into issues..  it could introduce a breaking change, right?

To prevent that, what do you think about improving the multiple server detection logic? For example, we could check if the dictionary contains keys whose values are also dictionaries (or have nested keys), which would suggest a multi-server config.